### PR TITLE
add wikidata and wikipedia for amenity/bank|ING

### PR DESCRIPTION
**Pull request by endorama**
ING is a Dutch multinational banking and financial services corporation.

[ING Group website](https://www.ing.com/Home.htm)  
[wikipedia](https://en.wikipedia.org/wiki/ING_Group)  
[wikidata](https://www.wikidata.org/wiki/Q645708)  

Thank you!

### CODE REVIEW

File: config/canonical.json [hunk 1]
```diff
@@ -2452,6 +2452,8 @@
     "tags": {
       "amenity": "bank",
       "brand": "ING",
+      "brand:wikidata": "Q645708",
+      "brand:wikipedia": "en:ING_Group",
```
Comment (bhousel) [inline config/canonical.json]: Can you change this to remove the underscore?  Thanks!
```suggestion
      "brand:wikipedia": "en:ING Group",
```

Comment (bhousel) [reply config/canonical.json]: Yes - technically what OpenStreetMap likes in this value is called a [local interlanguage link](https://en.wikipedia.org/wiki/Help:Interlanguage_links#Local_links).  The format is just the wiki language code, a colon, and the page title (underscores in the url become spaces in the title).

Comment (endorama) [reply config/canonical.json]: Thank you, nice to know that! Updated right now.

**Comment by bhousel** [comment]
Thank you @endorama 👍

**Pull request merged**